versioncontrol/peer.go

```diff
@@ -21,6 +21,8 @@ import (
 type Config struct {
 	Address  string `user:"true" help:"public address to listen on" default:":8080"`
 	Versions ServiceVersions
+
+	Version Versions
```

Comment:
All the "versions" are confusing me :).  What if we change this to `Binary Versions` and then update the properties to something like:
```
# URL for specific binary
binary.bootstrap.minimum.url: ""

# peer version
binary.bootstrap.minimum.version: v0.0.1
```

Reply:
sounds good!